.code-samples.meilisearch.yaml

```
@@ -868,8 +868,14 @@ update_faceting_settings_1: |-
     -X PATCH 'http://localhost:7700/indexes/books/settings/faceting' \
     -H 'Content-Type: application/json' \
     --data-binary '{
-      "maxValuesPerFacet": 2
-    }'
+      "maxValuesPerFacet": 2,
+      {
+        "sortFacetValuesBy": {
+          "*": "alpha",
+          "genres": count
+        }
+      }
```

```suggestion
      "sortFacetValuesBy": {
            "*": "alpha",
            "genres": count
      }
```
I may be wrong, but I'm surprised that this field is directly in an array.

---

Thanks for catching this. I fixed all of them